ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -1,9 +1,27 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+  ocp_builder_rhel-8-golang-1.18-openshift-4.12:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.18-openshift-4.12
 binary_build_commands: make manager
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.18
+images:
+- dockerfile_path: Dockerfile.must-gather
+  to: must-gather
+promotion:
+  additional_images:
+    kernel-module-management-must-gather: must-gather
```

Comment:
Isn't all the images under `images` being promoted by default?

Reply:
Could be. But by doing this, I suppose we can rename the resulting must-gather image in the ImageStream.

Reply:
So why don't we just call it `kernel-module-management-must-gather` instead of  `must-gather` in line 18 to begin with?